lib/ExpensiMark.js

```diff
@@ -959,27 +983,4 @@ export default class ExpensiMark {
         const linksInNew = this.extractLinksInMarkdownComment(newComment);
         return linksInOld === undefined || linksInNew === undefined ? [] : _.difference(linksInOld, linksInNew);
     }
-
-    /**
-     * Replace MD characters with their HTML entity equivalent
-     * @param {String} text
-     * @return {String}
-     */
-    escapeMarkdownEntities(text) {
-        // A regex pattern matching special MD characters we'd like to escape
-        const pattern = /([*_{}[\]~])/g;
-
-        // A map of MD characters to their HTML entity equivalent
-        const entities = {
-            '*': '&ast;',
-            _: '&lowbar;',
-            '{': '&lbrace;',
-            '}': '&rbrace;',
-            '[': '&lbrack;',
-            ']': '&rbrack;',
-            '~': '&#126;',
-        };
-
-        return text.replace(pattern, char => entities[char] || char);
-    }
```

Comment:
With the introduction of `attributeCleanup` rule - this logic is no longer needed